core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/site/edit/BottomNavPermissionConstants.kt

```diff
@@ -0,0 +1,14 @@
+package com.ustadmobile.core.viewmodel.site.edit
+import com.ustadmobile.core.MR
+import com.ustadmobile.core.db.BottomNavVisibilityFlags
+
+object BottomNavPermissionConstants {
+
+    val BOTTOM_NAV_LABELS = listOf(
```

Comment:
Should be removed as per comments on flags

Reply:
Add a new file: SiteConstants (in com.ustadmobile.viewmodel.site ). 

That should have a list of the top level items info that is common (e.g. the route name, the label string constant, the flag constant).

Then on each platform - create a list of those common constants plus the icon 

```
val iconMap = mapOf(
 flag to icon
) 
val TOP_LEVEL = commonTopLevel.map { ...
  it to iconMap[it.flag]
}
```